source/features/highlight-collaborators-and-own-conversations.tsx

```diff
@@ -53,6 +49,10 @@ void features.add({
 	include: [
 		pageDetect.isRepoConversationList
 	],
+	exclude: [
+		() => select.exists('.blankslate')
```

Comment:
This won't be found if we don't await dom ready, so it can be removed

Reply:
Tested, It works before dom ready

Reply:
Race condition warning‼️

If you're ok with it working sometimes and don't come back in 3 days with another PR that's fine. Of course this expects half the page to be loaded, so it's not gonna work 100% of the time

Reply:
Either way, there's no error.

Best case scenario, it avoids an API call.
Worst case scenario, there's an unnecessary API call.